examples/launchpad/launchpad.py

```diff
@@ -0,0 +1,39 @@
+from polygon import RESTClient
+from polygon.rest.models.request import RequestOptionBuilder
+
+
+def get_aggs_launchpad():
+    client = RESTClient()
+
+    """
+    options can be added to the RequestOptionBuilder both directly in 
+    initialization 
+    Example:
+    `options = RequestOptionBuilder(edge_id="", edge_ip_address="")
+    
+    or you can use the builder patten for future modifications to 
+    underlying edge header dictionary.
+    Example:
+    options = RequestOptionBuilder()
+        .edge_headers(edge_id="EDGE_ID", edge_ip_address="EDGE_ID_ADDRESS")
+        .update(edge_id="NEW")
+    options = options.update_edge_header(edge_ip_address="NEW_IP")
+    """
+    options = (
+        RequestOptionBuilder()
+        .edge_headers(edge_id="EDGE_ID", edge_ip_address="EDGE_ID_ADDRESS")
+        .update_edge_header(edge_user="EDGE_USER")
+    )
+
```

Comment:
I think we want two examples. One for setting everything in `edge_headers` and the other where we update. I fear that users might see this one example as the only way to do it

Reply:
"""
    set headers example:
    options = RequestOptionBuilder()
        .edge_headers(edge_id="EDGE_ID", edge_ip_address="EDGE_ID_ADDRESS")
        
    update headers example:
    options = options.update_edge_header(edge_ip_address="NEW_IP")
    """

Reply:
I like that. Can the first example add the User agent?

Reply:
I don't see this change incorporated.